tests/job_applications/test_transfer.py

```diff
@@ -278,9 +281,10 @@ def test_transfer_notifications_to_many_employers(self):
         job_application.transfer_to(origin_user_1, target_company)
 
         # Only checking SIAE email
-        assert len(mail.outbox) == 2
-        assert len(mail.outbox[0].to) == 2
-        assert origin_user_1.email in mail.outbox[0].to
-        assert origin_user_2.email in mail.outbox[0].to
+        assert len(mail.outbox) == 3
+        assert set(mail.outbox[0].to).isdisjoint(mail.outbox[1].to)
+        assert set(mail.outbox[0].to) < set([origin_user_1.email, origin_user_2.email])
+        assert set(mail.outbox[1].to) < set([origin_user_1.email, origin_user_2.email])
```

Comment:
Plus lisible, plus rapide, et vérifie qu’il n’y a qu’un seul destinataire par email. :shrug: 
```suggestion
        [first_mail_to] = mail.outbox[0].to
        [second_mail_to] = mail.outbox[1].to
        assert first_mail_to != second_mail_to
        assert first_mail_to in [origin_user_1.email, origin_user_2.email]
        assert second_mail_to in [origin_user_1.email, origin_user_2.email]
```